tests/_fixtures.py

```diff
@@ -68,6 +68,19 @@
      email STRING(MAX),
      deleted BOOL NOT NULL )
      PRIMARY KEY(id, commit_ts DESC);
+
+CREATE TABLE Customers (
```

Comment:
Why are we adding Customers table here when we already have `FKDAC_DDL` to add customers table.

Same for PG